packages/ember/tests/helpers/link_to_test.js

```diff
@@ -18,7 +19,7 @@ import { Application } from 'ember-application';
 import { jQuery } from 'ember-views';
 import { compile } from 'ember-template-compiler';
 import { setTemplates, setTemplate } from 'ember-glimmer';
-import { EMBER_IMPROVED_INSTRUMENTATION } from 'ember-debug';
```

Comment:
Pretty sure this is just wrong, and the tests in this file controlled by this value were not running.